### PR TITLE
debugger.md: Update docs for 1.3 features

### PR DESCRIPTION
**Pull request by gregg-miskelly**

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
To see the formatted view: https://github.com/gregg-miskelly/omnisharp-vscode/blob/DebuggerDocUpdates1.3/debugger.md

@rajkumar42 @chuckries please review

File: debugger.md [hunk 1]
```diff
@@ -140,6 +140,18 @@ The target process can optionally launch into a separate console window. You wil
...
+The debugger steps over properties and operators in managed code by default. In most cases, this provides a better debugging experience. To change this and enable stepping into properties or operators add:
+
+    "enableStepFiltering": false
+
+#### Attach Support
+The C# debugger supports attaching to processes. To do this, switch to the Debug tab, and open the configuration drop down.
+
+![Debug launch configuration drop down](https://raw.githubusercontent.com/wiki/OmniSharp/omnisharp-vscode/images/debug-launch-configurations.png)
+
+Select the '.NET Core Attach' configuration. Hitting the play button (or pressing F5) will then try and attach. In launch.json, if `processId` is set to `"${command.pickProcess}"` this will provide UI to select which process to attach to.
```
Comment (DustinCampbell) [inline debugger.md]: "Clicking" rather than "Hitting"?

"try to attach" rather than "try and attach"

**Comment by DustinCampbell** [comment]
:+1:

**Comment by chuckries** [comment]
:+1:

**Comment by gregg-miskelly** [comment]
@DustinCampbell I implemented your suggested changes

**Pull request merged**